consul/src/main/java/com/linecorp/armeria/server/consul/ConsulUpdatingListenerBuilder.java

```diff
@@ -139,6 +142,26 @@ public ConsulUpdatingListenerBuilder endpoint(Endpoint endpoint) {
         return this;
     }
 
+    /**
+     * Adds a tag to the list of tags associated with the service on registration.
+     *
+     * @param tag the tag to add
+     */
+    public ConsulUpdatingListenerBuilder addTag(String tag) {
```

Comment:
How about:

- Removing `addTag(String)`;
- Renaming `addTags(String...)` to `tags(String...)`; and
- Adding `tags(Iterable<String>)` ?

Armeria API usually doesn't have `add` prefix for additive properties. Everything is additive unless specified differently.